portal/src/main/scala/cromwell/pipeline/datastorage/dao/entry/UserEntry.scala

```diff
@@ -31,4 +31,6 @@ class UserEntry(val pipelineDatabaseEngine: PipelineDatabaseEngine) {
 
   def addUserAction(user: User) = (users.returning(users.map(_.userId))) += user
 
+  def updateUserById(updatedUser: User) = users.filter(_.userId ===  updatedUser.userId).update(updatedUser)
```

Comment:
If I'm correct, you will remove `profilePicture` during update if it previously existed.

Reply:
Great notice :)

@Dmitrii-Shevchenko, the updating process is more complex than it might seem at first.
The common user info (first name, last name, etc.), password and profile picture should be considered as different update queries.

Reply:
done